src/js/themes/hpe.js

```diff
@@ -1824,14 +1825,34 @@ const buildTheme = (tokens, flags) => {
         },
       },
     },
+    nameValueList: {
+      gap: {
+        column: 'xlarge',
+        row: 'xsmall',
+      },
+      pair: {
+        column: {
+          gap: {
+            column: 'xlarge',
+            row: 'medium',
+          },
+        },
+      },
+    },
     nameValuePair: {
       name: { color: 'text-strong', weight: global.hpe.fontWeight.medium },
     },
     notification: {
```

Comment:
Looks like we are missing re-mapping `notification.gap` along with some other notification props